src/golive/steps/cloudflare.md

```diff
@@ -0,0 +1,35 @@
+# (Optional) Cloudflare configuration
+
+One of the main features that a modern DNS provider needs to have in order to work well with Platform.sh is somethat that's colloquially known as "Cname Flattening".  This solves the problem of being able to point your "root domain" (example.com) to a domain name (cname) rather than an IP address (A record).  [This post](https://blog.cloudflare.com/introducing-cname-flattening-rfc-compliant-cnames-at-a-domains-root/) explains it well.
+
+In order to correctly point DNS to your Platform.sh project, you'll need at the very least the master environment cname, in other words the domain of your site before you add a custom domain in the project admin UI (or otherwise in the CLI).  It will look something like 
+
+TODO - get example
```

Comment:
I was just discussing this with @Fidelix in channel.  Shouldn't this point back to the main GoLive docs where we talk about getting that domain?  (Which is currently harder than it should be, but if we're going to change anything in docs for it we should do so centrally.)